__tests__/Schema.test.js

```diff
@@ -0,0 +1,92 @@
+/*eslint-disable-next-line*/
+const SchemaValidator = require('../lib/Schema');
+const errors = require('../lib/Errors');
+
+describe('Schema', () => {
+
+  const personSchema = {
+    firstName: { type: 'string', require: true },
+    lastName: { type: 'string', required: true },
+    married: { type: 'boolean' },
+    kids: { type: 'number' },
+  };
+
+  const schemaValidator = new SchemaValidator(personSchema);
+
+  const validModel = {
+    firstName: 'Dylan',
+    lastName: 'Corvidae',
+    married: true,
+    kids: 1,
+  };
+
+  const semiValidModel = {
+    firstName: 'Dylan',
+    lastName: 'Corvidae',
+    married: 'true',
+    kids: 1,
+  };
+
+  const invalidModel = {
+    firstName: [12, 16, 20, 24],
+    lastName: 'Corvidae',
+    married: 'purple',
+    kids: 1,
+  };
+
+  it('validates a valid model', () => {
+    expect(schemaValidator.validate(validModel)).toEqual(validModel);
+  });
+
+  it('validates a semivalid model', () => {
+    expect(schemaValidator.validate(semiValidModel)).toEqual(validModel);
+  });
+
+  it('throws on an invalid model', () => {
+    expect(() => {
+      schemaValidator.validate(invalidModel);
+    }).toThrow(errors.ModelError);
+  }); 
+}); 
+
+
+
+
+
+
+//Marty's Demo
```

Comment:
for labs moving forward, please do one last commit when you're finished where you remove the comments - they will still be in your previous commit on GitHub so you'll still be able to reference them.